certified-connectors/Data247logic/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3474 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Data247",
+    "description": "Welcome to the Data247 API! You can use our API to access the various data services we provide.",
+    "version": "1.0",
+    "contact":{
```

Comment:
The contact information should not contain PII. Is there a general support contact you can point users to instead?